src/test/java/study/StringCalculator.java

```diff
@@ -0,0 +1,97 @@
+package study;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.*;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringCalculator {
+
+    int calculator(String str) {
+        String[] cal_element = str.split(" ");
+        int num1 = Integer.parseInt(cal_element[0]);
+
+        for(int i=1; i<cal_element.length-1; i+=2) {
+            switch (cal_element[i]) {
+                case "+":
+                    num1 += Integer.parseInt(cal_element[i+1]);
+                    break;
+                case "-":
+                    num1 -= Integer.parseInt(cal_element[i+1]);
+                    break;
+                case "*":
+                    num1 *= Integer.parseInt(cal_element[i+1]);
+                    break;
+                case "/":
+                    num1 = (int)(num1/Integer.parseInt(cal_element[i + 1]));
+                    break;
+                default:
+                    throw new IllegalArgumentException();
+            }
+        }
+        return num1;
+    }
+
+    // 덧셈 테스트
+    @ParameterizedTest
+    @CsvSource(value = {"3 + 4:7", "2 + 5 + 8:15", "3 + 5 + 2:10"}, delimiter = ':')
+    void plus(String calculation, int result) {
+        assertThat(calculator(calculation)).isEqualTo(result);
+    }
+
+    // 뺄셈 테스트
+    @ParameterizedTest
+    @CsvSource(value = {"3 - 2:1", "5 - 2 - 3 - 1:-1"}, delimiter = ':')
+    void minus(String calculation, int result) {
+        assertThat(calculator(calculation)).isEqualTo(result);
+    }
+
+    // 곱셈 테스트
+    @ParameterizedTest
+    @CsvSource(value = {"3 * 2:6", "4 * 2 * 30:240"}, delimiter = ':')
+    void multiply(String calculation, int result) {
+        assertThat(calculator(calculation)).isEqualTo(result);
+    }
+
+    // 나눗셈 테스트
+    @ParameterizedTest
+    @CsvSource(value = {"3 / 2:1", "120 / 5 / 7:3"}, delimiter = ':')
+    void division(String calculation, int result) {
+        assertThat(calculator(calculation)).isEqualTo(result);
+    }
+
+    // 입력 값이 빈 공백 문자일 경우 테스트
```

Comment:
주석 대신에 `@DisplayName`과 같은 어노테이션을 활용하시는건 어떨까요?


```suggestion
    @DisplayName("입력 값이 빈 공백 문자일 경우 테스트")
```

Reply:
넵! 다음과 같이 변경하겠습니다 감사합니다!